lib/individual/health.dart

```diff
@@ -1,182 +1,220 @@
-                      borderRadius: BorderRadius.circular(10.r),
-                    ),
-                    child: Column(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        crossAxisAlignment: CrossAxisAlignment.center,
-                        children: [
-                          Text(
-                            "Select Organization",
-                            style: TextStyle(fontWeight: FontWeight.w500, fontSize: 20.sp),
-                          ),
-                          // SizedBox(
-                          //   height: 10.h,
-                          // )
-                          //
-                          // children:[
+        designSize: const Size(428, 926),
+        builder: (context, child) => Scaffold(
+              body: SafeArea(
+                  child: SizedBox(
+                width: double.infinity,
+                child: Container(
+                    decoration: const BoxDecoration(
+                        gradient: LinearGradient(
+                      colors: [
+                        Color(0xFF6CAFB4),
+                        Colors.white,
+                        Colors.white,
+                        Color(0xFF6CAFB4),
+                      ],
+                      begin: Alignment.topCenter,
+                      end: Alignment.bottomCenter,
+                      tileMode: TileMode.clamp,
+                    )),
+                    child: Stack(
+                      children: [
+                        Container(
+                            child: Column(children: [
                           Padding(
-                            padding: EdgeInsets.fromLTRB(15.w, 0, 15.w, 0),
-                            child: SearchField(
-                              hint: "Enter Organization's name",
-                              suggestions: const [
-                                'organization 1',
-                                'organization 2',
-                                'organization 3',
-                                'organization 4',
-                                'organization 5',
-                                'organization 6',
-                              ],
-                              searchInputDecoration: InputDecoration(
-                                enabledBorder: OutlineInputBorder(
-                                  borderSide: BorderSide(color: Colors.grey, width: 2.w),
-                                  borderRadius: BorderRadius.circular(80.r),
-                                ),
-                                focusedBorder: OutlineInputBorder(
-                                  borderRadius: BorderRadius.circular(80.r),
-                                  borderSide: BorderSide(color: const Color.fromARGB(255, 137, 184, 189), width: 2.w),
+                            padding: EdgeInsets.all(5.sp),
+                            child: Row(children: [
+                              IconButton(
```

Comment:
What do you think of using AppBar to place these elements? :thinking: 
You can center the title and also place a leading iconbutton, and I believe that this will simplify the code greatly.